step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/DataSetArtefact.java

```diff
@@ -19,48 +19,19 @@
 package step.artefacts;
 
 import step.core.artefacts.Artefact;
-import step.core.dynamicbeans.ContainsDynamicValues;
 import step.core.dynamicbeans.DynamicValue;
-import step.core.entities.EntityManager;
-import step.core.entities.EntityReference;
-import step.datapool.DataPoolConfiguration;
 
-@Artefact(name="DataSet")
+@Artefact(name= DataSetArtefact.DATA_SET_ARTIFACT_NAME)
 public class DataSetArtefact extends AbstractForBlock {
 
-	private DynamicValue<String> item = new DynamicValue<String>("dataSet");
```

Comment:
Has this field been removed on purpose? this field corresponds to the field "Iterator handle" in the UI which is still in use

Reply:
This field is already defined in parent `AbstractForBlock` class. Looks like there is no reason to have the same private field here (and also it breaks the serialization/deserialization to yaml).

Reply:
Ah, didn't se that. Thanks